src/Main.java

```diff
@@ -1,61 +1,144 @@
+        System.out.println("В сутки" + " машина произвела " + MachinePerformanceInPerDay + " штук");
+
+        byte NumberOfDays = 3;
+        short inThreeDaysMinutes = (short) (NumberOfDays * perDayMinutes);
+        int MachinePerformanceinThreeDays = MachinePerformanceIn1Minutes * inThreeDaysMinutes;
+        System.out.println("За 3 дня" + " машина произвела " + MachinePerformanceinThreeDays + " штук");
+
+        byte totalDaysInOneMonth = 30;
+        int inOneMonthOnlyMinutes = totalDaysInOneMonth * perDayMinutes;
+        int inOneMonth = inOneMonthOnlyMinutes * MachinePerformanceIn1Minutes;
+        System.out.println("За 1 месяц" + " машина произвела " + inOneMonth + " штук");
+
+        System.out.println();
+        System.out.println("Задание 5");
+        byte totalColors = 120;
+        byte oneClassWhite = 2;
+        byte oneClassBrown = 4;
+        byte totalPaintsForOneClass = (byte) (oneClassBrown + oneClassWhite);
+        byte totalClass = (byte) (totalColors / totalPaintsForOneClass);
+        byte classWhite = (byte) (oneClassWhite * totalClass);
+        byte classBrown = (byte) (oneClassBrown * totalClass);
+        System.out.println("В школе, где " + totalClass + " классов, нужно " + classWhite + " банок белой краски и "
+                + classBrown + " банок коричневой краски");
+        System.out.println();
+        System.out.println("Задание 6");
+        byte bananas = 5;
+        byte weightOfOneBanana = 80;
+        short totalWeightBanana = (short) (bananas * weightOfOneBanana);
+
+        short TotalMilkInLiters = 200;
+        byte oneHundredGramsOfMilk = 100;
+        byte amountOfOneHundredGramsOfMilk = (byte) (TotalMilkInLiters / oneHundredGramsOfMilk);
+        byte oneHundredLitersInGams = 105;
+        short numberOfGramsOfMilk = (short) (amountOfOneHundredGramsOfMilk * oneHundredLitersInGams);
+
+        byte allIceCream = 2;
+        byte weightOfOneIceCream = 100;
+        short totalWeightCream = (short) (weightOfOneIceCream * allIceCream);
+
+        byte totalEggs = 4;
+        byte weightOfOneEgg = 70;
+        short weightOfEggs = (short) (totalEggs + weightOfOneEgg);
+
+        short mixtureGrams = (short) (totalWeightBanana + numberOfGramsOfMilk + totalWeightCream + weightOfEggs);
+        float kilogramСonversion = 0.001f;
+        float mixtureKilogram = (float) (mixtureGrams * kilogramСonversion);
+        System.out.println(mixtureGrams + " грамм");
+        System.out.println(mixtureKilogram + " килограмм");
+
+        System.out.println();
+        System.out.println("Задание 7");
+        byte needToReset = 7;
+        short conversionToGramWeight = 1000;
+        short needToResetGram = (short) (needToReset * conversionToGramWeight);
+        short athleteLittleLose = 250;
+        short athleteLoseLot = 500;
+        short LosesFewDays = (short) (needToResetGram / athleteLittleLose);
+        short LosesManyDays = (short) (needToResetGram / athleteLoseLot);
```

Comment:
```suggestion
        short losesFewDays = (short) (needToResetGram / athleteLittleLose);
        short losesManyDays = (short) (needToResetGram / athleteLoseLot);
```
названия переменных надо писать с маленькой буквы